build/ci/templates/dotnet6-mobile-install-windows.yml

```diff
@@ -1,36 +1,22 @@
 parameters:
-  DotNetVersion: '6.0.401'
-  UnoCheck_Version: '1.5.4'
+  UnoCheck_Version: '1.6.0'
   UnoCheck_Manifest: 'https://raw.githubusercontent.com/unoplatform/uno.check/34b1a60f5c1c51604b47362781969dde46979fd5/manifests/uno.ui.manifest.json'
 
 steps:
 
-  # Required until .NET 6 installs properly on Windows using UseDotnet
-  # using preview builds
-  #- powershell: |
-  #    $ProgressPreference = 'SilentlyContinue'
-  #    Invoke-WebRequest -Uri "https://dot.net/v1/dotnet-install.ps1" -OutFile dotnet-install.ps1
-  #    & .\dotnet-install.ps1 -Version ${{ parameters.DotNetVersion }} -InstallDir "$env:ProgramFiles\dotnet\" -Verbose
-  #    & dotnet --list-sdks
-  #  displayName: Install .NET ${{ parameters.DotNetVersion }}
-  #  errorActionPreference: stop
-  #  retryCountOnTaskFailure: 3
-  - task: UseDotNet@2
-    displayName: 'Use .NET Core SDK ${{ parameters.DotNetVersion }}'
-    condition: and(succeeded(), ne(variables.UnoDisableNet6Mobile, 'true'))
-    retryCountOnTaskFailure: 3
-    inputs:
-      packageType: sdk
-      version: ${{ parameters.DotNetVersion }}
-      includePreviewVersions: true
+  - script: |
+      cp global.json global-net7.json
+      cp global-net6.json global.json
+    displayName: Replace global.json with .NET 6
+
+  - template: dotnet-install.yml
```

Comment:
`dotnet-install.yml` will only install whatever version in `global.json`, so with the copy above, I don't think .NET 7 will be installed for .NET 6 mobile builds.

Reply:
The uno-check tool does install the .NET SDK, given the manifest it is provided. You can validate that in the step which runs uno-check.

Reply:
@jeromelaban I Uno.Check at 1.5.4 as it was previously in the last commit

Reply:
Ok, then let's take a look at what the build logs show, to make sure there's no net7.

Reply:
Yeah waiting for CI to finish.